lib/matplotlib/afm.py

```diff
@@ -174,16 +178,42 @@ def _parse_header(fh):
     raise RuntimeError('Bad parse')
 
 
+CharMetrics = namedtuple('CharMetrics', 'width, name, bbox')
+CharMetrics.__doc__ = """
+    Represents the character metrics of a single character.
+
+    Notes
+    -----
+    The fields do currently only describe a subset of character metrics
+    information defined in the AFM standard.
+    """
+CharMetrics.width.__doc__ = "The character width (WX)."
+CharMetrics.name.__doc__ = "The character name (N)."
+CharMetrics.bbox.__doc__ = """
+    The bbox of the character (B) as a tuple (*llx*, *lly*, *urx*, *ury*)."""
+
+
 def _parse_char_metrics(fh):
     """
-    Return a character metric dictionary.  Keys are the ASCII num of
-    the character, values are a (*wx*, *name*, *bbox*) tuple, where
-    *wx* is the character width, *name* is the postscript language
-    name, and *bbox* is a (*llx*, *lly*, *urx*, *ury*) tuple.
+    Parse the given filehandle for character metrics information and returns
+    the information as dicts.
+
+    It is assumed that the file cursor is on the line behind
```

Comment:
I'm not sure I understand this "behind" business. Should the file cursor not be pointing at the 'S'?

Reply:
`_parse_header` consumes lines up to and including `StartCharMetrics`. It then returns and the file handle is passed on to `_parse_char_metrics`. The latter interprets the contents of a char metrics block, but not the `StartCharMetrics` command itself.

This is maybe not 100% clean and intuitive, but good enough for the internal parser functions. The docstring just describes what the code is doing/expecting.

Reply:
OK, I think I understand now. When I saw 'line behind something', I thought it was a _different_ line from 'something'. I also don't think 'behind' is the right word here.

Since the cursor is in the middle of a line anyway, I would not mention a line, just: 'the file cursor is pointing after "StartCharMetrics".'